assets/src/dashboard/app/reducer/stories.js

```diff
@@ -132,9 +132,7 @@ function storyReducer(state, action) {
           return acc;
         }
         fetchedStoriesById.push(current.id);
-        acc[current.id] = reshapeStoryObject(action.payload.editStoryURL)(
-          current
-        );
+        acc[current.id] = reshapeStoryObject()(current);
```

Comment:
In companion with the other suggestion, this can be simplified to:

```suggestion
        acc[current.id] = reshapeStoryObject(current);
```